packages/sdk/odahuflow/sdk/clients/model.py

```diff
@@ -116,7 +117,7 @@ def __init__(self,
             client_id=client_id,
             client_secret=client_secret,
             non_interactive=True,
-            base_url=url,
```

Comment:
In other clients url giving to the class constructor is basicly base url, for example 'https://gke05.dev.odahu.org', so we just use this as Authenticator url
But in ModelClient case, we giving more specified url like 'https://gke05.dev.odahu.org/model/test-wine', so u cant just use it in Authenticator, u have to extract a hostname before